server/athenian/api/align/queries/metrics.py

```diff
@@ -95,16 +128,19 @@ def _triage_metrics(metrics: List[str]) -> Tuple[List[str], List[str], List[str]
             continue
         unidentified.append(metric)
     if unidentified:
-        raise ResponseError(InvalidRequestError(
-            pointer=f".{MetricParamsFields.metrics}",
-            detail=f"The following metrics are not supported: {', '.join(unidentified)}",
-        ))
+        raise ResponseError(
+            InvalidRequestError(
+                pointer=f".{MetricParamsFields.metrics}",
+                detail=f"The following metrics are not supported: {', '.join(unidentified)}",
+            )
+        )
     return pr_metrics, release_metrics, jira_metrics
 
 
-def _loginify_teams(teams: Sequence[Collection[int]],
-                    prefixer: Prefixer,
-                    ) -> List[Set[str]]:
+def _loginify_teams(
+    teams: Sequence[Collection[int]],
+    prefixer: Prefixer,
```

Comment:
Interesting, so here it reformatted as I'd expect. What's the difference with `resolve_metrics_current_values`?